pkg/syncer/admissionhandler/admissionhandler.go

```diff
@@ -60,6 +60,7 @@ var (
 	featureGateTKGSHaEnabled                  bool
 	featureGateVolumeHealthEnabled            bool
 	featureGateTopologyAwareFileVolumeEnabled bool
+	featureGateSnapshotQuotaM2Enabled         bool
```

Comment:
Can you rename it to featureGateStorageQuotaM2Enabled?

Reply:
good catch. updated code.